### DISCUSSION TITLE
✅ IMPLEMENTED: Query Inspector: Cannot Inspect step and interval of Prometheus queries

### DISCUSSION THREAD

**Discussion by kustodian** [grafana/grafana#44275] [Observability]
**What happened**:
Before Grafana used GET to query Prometheus from where you could see all query parameters. Now when inspecting Prometheus queries Grafana shows you POST parameters that are sent to the Grafana backend and not what it's sent to Prometheus. 
See the screenshot:
![image](https://user-images.githubusercontent.com/1949229/147890842-d7743777-c740-4cd1-969d-025a89dd9bf3.png)
As you can see we cannot see what is the `$__rate_interval` passed, as well as what is the `step` parameter of the Prometheus query which makes it very hard to debug some query problems because you cannot see what was passed to Prometheus.

**What you expected to happen**:
I expect to be able to see the Prometheus `step` parameter as well as the values of `$_rate_interval` (see the full PromQL query sent to Prometheus).

**How to reproduce it (as minimally and precisely as possible)**:
Just make any Prometheus query.

**Environment**:
- Grafana version: 8.3.3
- Data source type & version: Prometheus 5.0.0
- OS Grafana is installed on: CentOS 7
- User OS & Browser: ArchLinux/Firefox
- Grafana plugins:
- Others:

**Comment by gabor** [comment]
a similar feature request https://github.com/grafana/grafana/discussions/42079

**Comment by ivanahuckova** [comment]
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/30407135/196412393-9b9a1b57-9c1a-47b1-9d7f-77c9ec6303b8.png">
